closed/make/Main.gmk

```diff
@@ -27,7 +27,7 @@ CLEAN_DIRS += vm
 	#
 
 j9vm-build :
-	+($(CD) $(SRC_ROOT)/closed && $(MAKE) -f OpenJ9.gmk SPEC=$(SPEC) VERSION_MAJOR=$(VERSION_MAJOR) build-j9)
+	+($(CD) $(SRC_ROOT)/closed && $(MAKE) -f OpenJ9.gmk SPEC=$(SPEC) VERSION_MAJOR=$(VERSION_MAJOR) OPENJ9_BUILD=true build-j9)
```

Comment:
Not sure why this is needed. OpenJ9.mk exports `OPENJ9_BUILD=true` already and this was already working in the makefiles.

Reply:
That's right, it is not required. Verified by a personal build.
Closing this PR.